bin/express-cli.js

```diff
@@ -68,7 +76,7 @@ if (!exit.exited) {
  * Install an around function; AOP.
  */
 
-function around (obj, method, fn) {
```

Comment:
There are a lot of changes like this that don't seem related to yiur changes. In fact deleting the space is a linting error